examples/PrecompileCategoryOfSkeletalFinSets.g

```diff
@@ -12,6 +12,14 @@ given_arguments := [ ];;
 compiled_category_name := "CategoryOfSkeletalFinSetsPrecompiled";;
 package_name := "FinSetsForCAP";;
 
+CapJitAddLogicTemplate(
+    rec(
+        variable_names := [ "list", "func" ],
+        src_template := "List( list, func )[1]",
+        dst_template := "func( list[1] )",
```

Comment:
See this comment for the reason why we do not do this: https://github.com/homalg-project/FinSetsForCAP/pull/90#issuecomment-1032566037 The compiler has a rule in the opposite direction, so I'm actually surprised that this does not lead to an infinite recursion :D

Reply:
I keep forgetting these things :) Shall I remove the rule?

Reply:
Yes, please.